charts/jenkins/values.yaml

```
@@ -368,7 +368,7 @@ master:
   affinity: {}
   # Leverage a priorityClass to ensure your pods survive resource shortages
   # ref: https://kubernetes.io/docs/concepts/configuration/pod-priority-preemption/
-  # priorityClass: system-cluster-critical
+  # priorityClassName: system-cluster-critical
```

This would be even better.
```suggestion
  # priorityClassName: system-cluster-critical
  priorityClassName:
```